tests/components/volvooncall/test_config_flow.py

```diff
@@ -51,8 +52,10 @@ async def test_form_invalid_auth(hass: HomeAssistant) -> None:
     )
 
     with patch(
-        "homeassistant.components.volvooncall.config_flow.VolvoOnCallConfigFlow.is_valid",
-        side_effect=InvalidAuth,
+        "aiohttp.client_exceptions.ClientResponseError.__init__", return_value=None
```

Comment:
Why are we patching the aiohttp exception? Pass a Mock to the exception if we need to instead.

Reply:
Example:
https://github.com/home-assistant/core/blob/9e66b30af97063b500fb41a9c82a2565bf431082/tests/components/android_ip_webcam/test_config_flow.py#L112